docs/source/getting-started/first-feature.rst

```diff
@@ -72,7 +76,7 @@ TypeScript Code Generation
 
 Remember that we interface with the DAML model from the UI components using generated TypeScript.
 Since we have changed our DAML code, we also need to rerun the TypeScript code generator.
-Let's do this by running::
+Open a new terminal Let's do this by running::
```

Comment:
```suggestion
Open a new terminal and run the following commands::
```
It sounds like you’re explaining how to open a new terminal with those commands.